### ISSUE TITLE
Add Goose to https://github.com/denji/awesome-http-benchmark

### ISSUE THREAD

**Issue by kazimir-malevich** [tag1consulting/goose#476]
I was thinking:

"Goose: A modern and comprehensive HTTP load test tool built in Rust with distribution freely included"

**Comment by jeremyandrews** [comment]
Yes, it's a good plan to get into any lists out there! Are you going to submit a PR there, or shall i?

Reading through some of the other examples, perhaps copying the existing style more, something like:

[Goose](https://goose.rs/): Modern, high-performance and asynchronous distributed HTTP load test tool, written in Rust.

**Comment by kazimir-malevich** [comment]
I'll create the PR. It might also be an idea posting releases on the Rust subreddit and eventually creating a Wikipedia page.

**Comment by kazimir-malevich** [comment]
PR created:
https://github.com/denji/awesome-http-benchmark/pull/39/commits/448f7b91a27b560866dfd075e7af883867828e6c

**Comment by jeremyandrews** [comment]
We've definitely been posting to the Rust subreddit (and received a lot of useful feedback/help there, too):
https://www.reddit.com/r/rust/search/?q=goose

That said, it's been a while since we've posted an update and there's been a lot of improvements lately. I generally wait until we've written a blog with more insight into what's new/different.

A Wikipedia page would be great!

**Comment by kazimir-malevich** [comment]
PR has has been merged.
https://github.com/denji/awesome-http-benchmark/pull/39

Wikipedia is going to be a hard one, I think we might need more traction and at least v1. I'm sure we'll get there.

**Comment by dralley** [comment]
Some comparison against [Drill](https://github.com/fcsonline/drill) and [pdc](https://gitlab.com/liamwarfield/pdc) would be helpful, perhaps even collaboration?  For instance I could imagine Drill using Goose as a backend.
